ScratchDjango/User/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.db.models import CharField, EmailField
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from .managers import UserManager
+from .utils import check_email
+
+
+class User(AbstractUser):
+    # First and last name do not cover name patterns around the globe
+    name = CharField(_("Name of User"), blank=True, max_length=255)
```

Comment:
I don't think we need to add first_name, or last_name here. They are already added to the User model.